docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Enroll an agent into automatic updates
+description: How to enroll an agent into automatic updates
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  Automatic agent update is available starting from Teleport `13.0`.
+</Details>
+
+Starting with version 13, Teleport supports automatic agent updates for
```

Comment:
Nit: I think "starting with version 13" is redundant since you have a warning above.
```suggestion
Teleport supports automatic agent updates for
```